ngraph/test/runtime/interpreter/evaluates_map.cpp

```diff
@@ -300,6 +300,72 @@ namespace
         return true;
     }
 
+    template <element::Type_t ET>
+    bool evaluate(const shared_ptr<op::v6::MVN>& op,
+                  const HostTensorVector& outputs,
+                  const HostTensorVector& inputs)
+    {
+        using T = typename element_type_traits<ET>::value_type;
+        AxisSet reduction_axes;
+        auto rank = inputs[0]->get_shape().size();
+        if (inputs[1]->get_element_type() == element::i64)
```

Comment:
The code in both if branches is exactly the same. This should be extracted to a template function taking `int64_t` or `int32_t` as a template param.